guessingGame.js

```diff
@@ -0,0 +1,19 @@
+function GuessingGame(answer) {
+  this.answer = answer;
+  this.solved = false;
+  this.guess = function(guess) {
```

Comment:
Look into adding functions to a constructor prototype over adding within the constructor function.